module/strategy/Defend/src/Defend.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef MODULE_STRATEGY_DEFEND_HPP
+#define MODULE_STRATEGY_DEFEND_HPP
+
+#include <Eigen/Core>
+#include <nuclear>
+
+#include "extension/Behaviour.hpp"
+
+namespace module::strategy {
+
+    class Defend : public ::extension::behaviour::BehaviourReactor {
+    private:
+        /// @brief Stores configuration values
+        struct Config {
+
+            /// @brief Region to defend the ball
```

Comment:
this isn't really clear what the 4 dimensional vector means. If these are min and max values it would make sense to split them up into their own variables. They aren't a 4 dimensional vector, they are four separate values.